plugin/route53/README.md

```diff
@@ -18,6 +18,7 @@ route53 [ZONE:HOSTED_ZONE_ID...] {
     aws_access_key [AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY]
     credentials PROFILE [FILENAME]
     fallthrough [ZONES...]
+    frequency SECONDS
```

Comment:
Maybe call this `refresh`? `frequency` seems too generic...

Reply:
Why not use duration string here instead? Example:
```
  refresh 1s
```

Then you can use `time.ParseDuration` in the setup and skip all this strconv boilerplate.

Reply:
Updated and I think I got it as simple as I could while still considering numbers without units as seconds to keep it reasonable.